tool/flutter_ci_script_shared.sh

```diff
@@ -12,11 +12,7 @@ function ci_projects () {
         flutter pub get
 
         # Run the analyzer to find any static analysis issues.
-        if [ "$channel" == 'stable' ]; then
-            dart analyze --fatal-infos
-        else
-            dart analyze
-        fi
+        dart analyze --fatal-infos
```

Comment:
Something tells me this is the primary part of interest

Reply:
Yup, thats the real change